<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceCoverArtBoxOptionsPage</class>
 <widget class="QWidget" name="InterfaceCoverArtBoxOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>200</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="cb_show_cover_art_details">
     <property name="title">
      <string>Show cover art details in cover art view</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="cover_art_details_layout">
      <item>
       <widget class="QCheckBox" name="cb_show_cover_art_details_type">
        <property name="text">
         <string>Show type (front, back, etc.)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_show_cover_art_details_filesize">
        <property name="text">
         <string>Show file size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_show_cover_art_details_dimensions">
        <property name="text">
         <string>Show dimensions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_show_cover_art_details_mimetype">
        <property name="text">
         <string>Show MIME type</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
